lib/liquid_pull_to_refresh.dart

```diff
@@ -47,19 +47,18 @@ enum _LiquidPullToRefreshMode {
 class LiquidPullToRefresh extends StatefulWidget {
   const LiquidPullToRefresh({
     Key key,
+    this.animSpeedFactor = 1.0,
     @required this.child,
     @required this.onRefresh,
     this.color,
     this.backgroundColor,
-    this.notificationPredicate = defaultScrollNotificationPredicate,
     this.height,
     this.springAnimationDurationInMilliseconds = 1000,
     this.borderWidth = 2.0,
     this.showChildOpacityTransition = true,
-    this.scrollController,
   })  : assert(child != null),
         assert(onRefresh != null),
-        assert(notificationPredicate != null),
+        assert(animSpeedFactor > 1.0),

```

Comment:
@AadumKhor 
Don't you think it should be `animSpeedFactor >= 1.0`?

Reply:
Yeah. My bad.